alarmee/src/androidMain/kotlin/com/tweener/alarmee/AlarmeeSchedulerAndroid.kt

```diff
@@ -107,6 +107,37 @@ class AlarmeeSchedulerAndroid(
         }
     }
 
+    override fun pushNotificationNow(alarmee: Alarmee, onSuccess: () -> Unit) {
+        createNotificationChannels(context = context)
+        validateNotificationChannelId(alarmee)
+
+        val notification = alarmee.androidNotificationConfiguration.channelId?.let {
+            NotificationCompat.Builder(
+                context,
+                it
+            )
+                .apply {
+                    setSmallIcon(configuration.notificationIconResId)
+                    setContentTitle(alarmee.notificationTitle)
+                    setContentText(alarmee.notificationBody)
+                    setPriority(mapPriority(alarmee.androidNotificationConfiguration.priority))
+                    setAutoCancel(true)
+                }
+                .build()
+        }
+
+        context.getNotificationManager()?.let { notificationManager ->
+            if (notificationManager.areNotificationsEnabled()) {
+                notificationManager.notify(alarmee.uuid.hashCode(), notification)
```

Comment:
`notification` here can be null if `channelId` was null previously. And according to the documentation of the `notificationManager.notify()` method, it should not be null:
```notification – A Notification object describing what to show the user. Must not be null.```.

Can you update this part of the code to avoid that? Thanks!